### DISCUSSION TITLE
[Bug]: Small character defects cannot be predicted

### DISCUSSION THREAD

**Discussion by xingfenghaizeiwang** [open-edge-platform/anomalib#1296] [Q&A]
### Describe the bug

![11e9a62fc58cc20988bdef40c0613ab](https://github.com/openvinotoolkit/anomalib/assets/104884533/f0be038f-bb4b-46d4-86ac-cd090830325b)

### Dataset

Folder

### Model

PADiM

### Steps to reproduce the behavior

Run python tools/train.py --config D:\project\anomalib\src\anomalib\models\padim\config.yaml

### OS information

Windows 10:
- Python version: [ 3.9.10]
- Anomalib version: [ 0.6.0]
- PyTorch version: [1.13.1]
- CUDA/cuDNN version: [11.7]
- GPU models and configuration: [GeForce RTX 3060]

### Expected behavior

Small character defects can be predicted

### Screenshots

![11](https://github.com/openvinotoolkit/anomalib/assets/104884533/e5de3ff6-1cfa-4ceb-beb1-8e0d61d76c51)
![12](https://github.com/openvinotoolkit/anomalib/assets/104884533/e37f9d0a-403e-4b75-87f3-09cbcfe71271)

### Pip/GitHub

pip

### Configuration YAML

```yaml
dataset:
  name: mvtec
  format: folder
  root: D:\project\anomalib\datasets\MVTec
  normal_dir: make/train/good
  normal_test_dir: make/test/good
  abnormal_dir: make/test/ng
  task: segmentation
  mask_dir: null
  extensions: null
  train_batch_size: 32
  eval_batch_size: 32
  num_workers: 8
  image_size: 256 # dimensions to which images are resized (mandatory)
  center_crop: null # dimensions to which images are center-cropped after resizing (optional)
  normalization: imagenet # data distribution to which the images will be normalized: [none, imagenet]
  transform_config:
    train: null
    eval: null
  test_split_mode: none # options: [from_dir, synthetic, none]
  test_split_ratio: 0.2 # fraction of train images held out testing (usage depends on test_split_mode)
  val_split_mode: synthetic # options: [same_as_test, from_test, synthetic]
  val_split_ratio: 0.5 # fraction of train/test images held out for validation (usage depends on val_split_mode)
  tiling:
    apply: false
    tile_size: null
    stride: null
    remove_border_count: 0
    use_random_tiling: False
    random_tile_count: 16

model:
  name: padim
  backbone: wide_resnet50_2
  pre_trained: true
  layers:
    - layer1
    - layer2
    - layer3
  normalization_method: min_max # options: [none, min_max, cdf]

metrics:
  image:
    - F1Score
    - AUROC
  pixel:
    - F1Score
    - AUROC
  threshold:
    method: adaptive #options: [adaptive, manual]
    manual_image: null
    manual_pixel: null

visualization:
  show_images: False # show images on the screen
  save_images: True # save images to the file system
  log_images: True # log images to the available loggers (if any)
  image_save_path: null # path to which images will be saved
  mode: full # options: ["full", "simple"]

project:
  seed: 42
  path: ./result

logging:
  logger: [ ] # options: [comet, tensorboard, wandb, csv] or combinations.
  log_graph: false # Logs the model graph to respective logger.

optimization:
  export_mode: null # options: torch, onnx, openvino

# PL Trainer Args. Don't add extra parameter here.
trainer:
  enable_checkpointing: true
  default_root_dir: null
  gradient_clip_val: 0
  gradient_clip_algorithm: norm
  num_nodes: 1
  devices: 1
  enable_progress_bar: true
  overfit_batches: 0.0
  track_grad_norm: -1
  check_val_every_n_epoch: 1 # Don't validate before extracting features.
  fast_dev_run: false
  accumulate_grad_batches: 1
  max_epochs: 1
  min_epochs: null
  max_steps: -1
  min_steps: null
  max_time: null
  limit_train_batches: 1.0
  limit_val_batches: 1.0
  limit_test_batches: 1.0
  limit_predict_batches: 1.0
  val_check_interval: 1.0 # Don't validate before extracting features.
  log_every_n_steps: 50
  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
  strategy: null
  sync_batchnorm: false
  precision: 32
  enable_model_summary: true
  num_sanity_val_steps: 0
  profiler: null
  benchmark: false
  deterministic: false
  reload_dataloaders_every_n_epochs: 0
  auto_lr_find: false
  replace_sampler_ddp: true
  detect_anomaly: false
  auto_scale_batch_size: false
  plugins: null
  move_metrics_to_cpu: true
  multiple_trainloader_mode: max_size_cycle
```

### Logs

```shell
no save
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samet-akcay** [comment]
Hi @xingfenghaizeiwang, loss is `None` because PADIM does not have any optimisation process requiring a loss computation. For more details you could refer the paper
https://arxiv.org/abs/2011.08785

**Comment by samet-akcay** [comment]
Since this is not a bug, I'm converting this to a Q&A discussion.
